src/client/components/pages/revisionsListPage.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2018 Akhilesh Kumar <akhilesh5991@gmail.com>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import * as bootstrap from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import React from 'react';
+import RevisionsTable from './parts/revisions-diplayTable';
+import request from 'superagent-bluebird-promise';
+
+
+const {Pager, Button, ButtonGroup, DropdownButton, MenuItem} = bootstrap;
+
+class RevisionsListPage extends React.Component {
```

Comment:
Let's replace wherever you have 'RevisionsList' everywhere in the various files with 'Revisions'  (including 'RevisionsListPage' -> 'RevisionsPage')